Source/BluemixAppID/api/AppID.swift

```diff
@@ -37,15 +38,19 @@ public class AppID {
         self.bluemixRegion = bluemixRegion
 		self.oauthManager = OAuthManager(appId: self)
         self.loginWidget = LoginWidgetImpl(oauthManager: self.oauthManager!)
+        self.userAttributeManager = UserAttributeManagerImpl(appId: self)
     }
 	
     public func loginAnonymously(authorizationDelegate:AuthorizationDelegate) {
-        // TODO: we need to complete this
-        self.loginAnonymously(accessToken: nil, authorizationDelegate: authorizationDelegate)
+        self.loginAnonymously(accessTokenString: nil, authorizationDelegate: authorizationDelegate)
     }
     
-    public func loginAnonymously(accessToken:String?, authorizationDelegate:AuthorizationDelegate) {
-        // TODO: we need to complete this
+    public func loginAnonymously(accessTokenString:String?, authorizationDelegate:AuthorizationDelegate) {
+        self.loginAnonymously(accessTokenString: accessTokenString, allowCreateNewAnonymousUsers: true, authorizationDelegate: authorizationDelegate)
+    }
+    
+    public func loginAnonymously(accessTokenString:String?, allowCreateNewAnonymousUsers: Bool, authorizationDelegate:AuthorizationDelegate) {
```

Comment:
No need to have three signatures. You can use Swift optional arguments with default values. Talked to @motyd about this.